packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -258,25 +235,43 @@ export class FauxClassGenerator {
     }
 
     if (!this.createIfNeededOutputFolder(standardSObjectsFolderPath)) {
-      throw nls.localize('no_sobject_output_folder_text', standardSObjectsFolderPath);
+      throw nls.localize(
+        'no_sobject_output_folder_text',
+        standardSObjectsFolderPath
+      );
     }
 
     const sobjectDecl: SObjectDefinition[] = this.getSObjectSubsetDefinitions();
     this.generateAndWriteFauxClasses(sobjectDecl, standardSObjectsFolderPath);
     this.result.data.standardObjects = sobjectDecl.length;
     this.logSObjects('Standard', sobjectDecl.length);
+
+    try {
+      this.typingGenerator.generate(sobjectDecl, typingsFolderPath);
+    } catch (errorMessage) {
+      return this.errorExit(errorMessage);
+    }
+
     return this.successExit();
   }
 
   // VisibleForTesting
-  public generateAndWriteFauxClasses(sobjectDecl: SObjectDefinition[], standardSObjectsFolderPath: string) {
+  public generateAndWriteFauxClasses(
+    sobjectDecl: SObjectDefinition[],
+    standardSObjectsFolderPath: string
+  ) {
     // This method is different from generateFauxClasses -  generateFauxClasses takes SObject array as input
     // and to generate that we would need a large definition file. If we go one level more specific, as what
     // generateAndWriteFauxClasses here requires, simpler declarations we have in the minSObjects.json is good.
     for (const sobject of sobjectDecl) {
-      const fauxClassPath = path.join(standardSObjectsFolderPath, sobject.name + '.cls');
-      sobject.fields.forEach(field => { field.modifier = 'global'; });
-      fs.writeFileSync(fauxClassPath, this.generateFauxClassTextFromDecls(sobject.name, sobject.fields), {
+      const fauxClassPath = path.join(
+        standardSObjectsFolderPath,
+        sobject.name + APEX_CLASS_EXTENSION
```

Comment:
```suggestion
        `${sobject.name}${APEX_CLASS_EXTENSION}`
```

Reply:
I did this change directly in GitHub so please check that it does work before merging